Rakefile

```diff
@@ -45,8 +45,8 @@ end
 task :default => :test
 
 RDoc::Task.new do |rdoc|
-  rdoc.title = "IRB"
-  rdoc.rdoc_dir = "docs"
-  rdoc.main = "README.md"
-  rdoc.options.push("--copy-files", "LICENSE.txt")
+  rdoc.title = "IRB Documentation"
+  rdoc.main = "index.md"
+  rdoc.rdoc_dir = "_site"
+  rdoc.options.push("lib")
```

Comment:
We also need this task because we now need to use `lib` as the target folder, not the root folder.